package-lock.json

```diff
@@ -1872,6 +1873,27 @@
         "node": ">=12"
       }
     },
+    "node_modules/cli-cursor": {
+      "version": "3.1.0",
+      "resolved": "https://artifactory.global.square/artifactory/api/npm/square-npm/cli-cursor/-/cli-cursor-3.1.0.tgz",
```

Comment:
Sanity: are you sure all these tens of `https://artifactory.global.square` square specific reference won't break install? They break for me.

Reply:
Ah, sorry didn't realize it was adding artifactory stuff. In any case, I removed the package from this PR